frontend/public/components/deployment-config.tsx

```diff
@@ -207,22 +256,38 @@ const DeploymentConfigTableHeader = () => {
 };
 DeploymentConfigTableHeader.displayName = 'DeploymentConfigTableHeader';
 
-export const DeploymentConfigsList: React.FC = props => <Table {...props} aria-label="Deployment Configs" Header={DeploymentConfigTableHeader} Row={DeploymentConfigTableRow} virtualize />;
+export const DeploymentConfigsList: React.FC = (props) => (
+  <Table
+    {...props}
+    aria-label="Deployment Configs"
+    Header={DeploymentConfigTableHeader}
+    Row={DeploymentConfigTableRow}
+    virtualize
+  />
+);
 DeploymentConfigsList.displayName = 'DeploymentConfigsList';
 
-export const DeploymentConfigsPage: React.FC<DeploymentConfigsPageProps> = props => {
+export const DeploymentConfigsPage: React.FC<DeploymentConfigsPageProps> = (props) => {
   const createItems = {
-    image: 'From Image',
     yaml: 'From YAML',
   };
 
   const createProps = {
     items: createItems,
-    createLink: (type: string) => type === 'image'
-      ? `/deploy-image${props.namespace ? `?preselected-ns=${props.namespace}` : ''}`
-      : `/k8s/ns/${props.namespace || 'default'}/deploymentconfigs/~new`,
+    createLink: () => `/k8s/ns/${props.namespace || 'default'}/deploymentconfigs/~new`,
```

Comment:
This whole file got update with Prettier because I had to update this one function. This is the only place I intentionally made changes. 
@spadgett

Reply:
We shouldn't be running prettier on files in `/public` dir.